crane/container.go

```diff
@@ -51,16 +51,17 @@ type ContainerInfo interface {
 }
 
 type container struct {
-	id        string
-	RawName   string
-	RawUnique bool            `json:"unique" yaml:"unique"`
-	RawImage  string          `json:"image" yaml:"image"`
-	RawBuild  BuildParameters `json:"build" yaml:"build"`
-	RawRun    RunParameters   `json:"run" yaml:"run"`
-	RawRm     RmParameters    `json:"rm" yaml:"rm"`
-	RawStart  StartParameters `json:"start" yaml:"start"`
-	RawExec   ExecParameters  `json:"exec" yaml:"exec"`
-	hooks     hooks
+	id          string
+	RawName     string
+	RawUnique   bool            `json:"unique" yaml:"unique"`
+	RawImage    string          `json:"image" yaml:"image"`
+	RawRequires []string        `json:"requires" yaml:"requires"`
```

Comment:
I think it's actually fine here, `unique` is already a non-Docker value. So is `image`, if you think about it - it sort of belongs into the run config, but that has no option `image`. So I would say non-Docker keys at the container level are okay - as there is no equivalent "command" on the Docker side.